akita/cmd/builderTemplate.txt

```diff
@@ -0,0 +1,34 @@
+package {{packageName}}
+
+import "github.com/sarchlab/akita/v4/sim"
+
+type Builder struct {
```

Comment:
Could you add the documentation comments for the generated code? 

Say the builder can be `// Builder can create a {{packageName}} with different configurations.`

Reply:
Documentation comments created for builder and comp.